oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/FileCacheTest.java

```diff
@@ -108,6 +108,7 @@ public void zeroCache() throws Exception {
         assertNull(cache.getIfPresent(ID_PREFIX + 0));
         assertNull(cache.get(ID_PREFIX + 0));
         assertEquals(0, cache.getStats().getMaxTotalWeight());
+        assertEquals(0, cache.getStats().getElementCount());
```

Comment:
we should add assertion for other fields as well i.e. evictionCount, totalLoadTime etc.
All fields that throw an exception should be asserted.